app/bundles/FormBundle/Entity/Form.php

```diff
@@ -204,6 +209,11 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->nullable()
             ->build();
 
+        $builder->createField('inContactTab', 'boolean')
+            ->columnName('in_contact_tab')
+            ->nullable()
+            ->build();
```

Comment:
This could be a one liner. See for example https://github.com/mautic/mautic/blob/staging/app/bundles/WebhookBundle/Entity/Log.php#L73.